resources/ssl/README

```diff
@@ -1,5 +1,6 @@
 Files that should be placed in this directory for an SSL secured manager:
-server.key - The private key of the server.
-server.crt - The public certificate of the server.
 
-For more details, see: http://docs.getcloudify.org/3.4.0/manager/security/#ssl
+cloudify_external_key.pem - The private key of the server.
+cloudify_external_cert.pem - The public certificate of the server.
+
+For more details, see: http://docs.getcloudify.org/4.0.1/manager/security/#ssl
```

Comment:
I am also not sure. I need to check with Arthur. It might need to wait until he returns from overseas. In the meantime, I have approved.